test-apps/README.md

```diff
@@ -0,0 +1,3 @@
+This folder contains apps (services) designed for exercising test scenarios in Waiter.
+It also includes a script to create the docker image for use in the different schedulers.
+The docker image includes executables to the different test apps.
```

Comment:
Might be nice to include a sentence or two about each sub-directory (like "kitchen is for ..." and "nginx is for ...")